docker-compose/scripts/run.sh

```diff
@@ -23,6 +23,10 @@ fi
 
 cp -a /app/docker-compose/config/* /app/config/
 
+envsubst < /app/docker-compose/config/database.yml > /app/config/database.yml
+envsubst < /app/docker-compose/config/redis.yml > /app/config/redis.yml
+
+sleep 600
```

Comment:
Can you add a comment explaining why this ```sleep``` is here?

Reply:
its added for debug work. removed now